kubernetes_platform/proto/kubernetes_executor_config.proto

```diff
@@ -154,3 +154,11 @@ message ImagePullSecret {
     // Name of the image pull secret.
     string secret_name = 1;
 }
+
+message Toleration {
+    string key = 1;
+    string operator = 2;
+    string value = 3;
+    string effect = 4;
+    optional int64 toleration_seconds = 5;

```

Comment:
This needs to be optional otherwise not passing it results in it defaulting to 0 which leads to unintended effects.